src/star.cpp

```diff
@@ -1031,8 +1031,9 @@ bool GuideStar::AutoFind(const usImage& image, int extraEdgeAllowance, int searc
             // We're repeating the find, so we're vulnerable to hot pixels and creation of unwanted duplicates
             if (tmp.WasFound() && tmp.SNR >= minSNR)
             {
-                bool duplicate = std::find_if(foundStars.begin(), foundStars.end(), [&tmp](const GuideStar& other)
-                                              { return CloseToReference(tmp, other); }) != foundStars.end();
+                bool duplicate =
+                    std::find_if(foundStars.begin(), foundStars.end(),
+                                 [&tmp](const GuideStar& other) { return CloseToReference(tmp, other); }) != foundStars.end();

```

Comment:
this change seems to be unrelated to the pr ... let's revert it and get star.cpp out of the pr

Reply:
This is the same old problem with clang and star.cpp.   This has reverted the clang change without me ever opening or touching that file.  Something is seriously messed up with this.

Reply:
we'll need to find out what is changing that file in your local environment, and also be sure not to commit the file -- you can run `git restore src/star.cpp` before you commit it if you notice that it has been changed unintentionally.
Now that the "bad" version has been committed into the branch you can get the old version back with
```
git restore -s origin/master -- src/star.cpp
```
(or use any other version you want in place of `origin/master`)
Then `git add` `git commit` and `git push` to get the branch fixed up on github (origin).